notebooks/Add GPS Metadata Fields.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<p>Should this be "feature class" instead of "feature layer"? since it's working within Pro?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>